scripts/cicd/verify-multiaccount-records-have-been-ingested.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env -S python3 -u
```

Comment:
That's for moving this logic to a Python script. It's much more readable, easier to maintain and we can also run lint, style and other static analysis checks on it...